machine_learning_hep/analysis/analyzerdhadrons_mult.py

```diff
@@ -85,6 +85,9 @@ def __init__(self, datap, case, typean, period):
         self.p_bin_width = datap["analysis"][self.typean]['bin_width']
         self.p_num_bins = int(round((self.p_mass_fit_lim[1] - self.p_mass_fit_lim[0]) / \
                                     self.p_bin_width))
+        self.Nbx2 = datap["analysis"][self.typean].get("isNbx2","")
+        if self.Nbx2 not in datap["analysis"][self.typean]:
```

Comment:
Shouldn't this look like this?
`if "isNbx2" not in datap["analysis"][self.typean]:`